frog/imports/ui/GraphEditor/store/operator.js

```diff
@@ -90,10 +89,22 @@ export default class Operator {
     };
   }
 
-  @computed get dragPoint(): [number, number] {
+  @computed get dragPointTo(): [number, number] {
+    return [this.x + 30, this.y + 30];
```

Comment:
Is this 30 because the operator is 60px in diameter? Maybe leaving a comment here, could be useful.